src/styles/_catalog.scss

```diff
@@ -1,22 +1,35 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
 
   max-width: 944px;
   margin: 0 auto;
 
   &__card {
-    display: flex;
+    display: grid;
     justify-content: center;
     align-items: center;
     width: 200px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    gap: 48px;
```

Comment:
this gap doesn't work after breakpoints, I can't advise how to fix it, but check your code again
![image](https://user-images.githubusercontent.com/87998640/222364763-edc78fe1-8a52-4f66-ac7e-c3045cbf48dd.png)